vulnerabilities/migrations/0034_package_package.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.7 on 2022-11-25 12:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('vulnerabilities', '0033_alter_vulnerabilityseverity_scoring_system'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='package',
```

Comment:
This means you are storing the concrete purl, and this makes 100% sense. But then call it purl or pakage_url and replace/override anything we have already.